content/en/synthetics/private_locations/_index.md

```diff
@@ -86,15 +88,23 @@ To pull test configurations and push test results, the private location worker n
 
 {{< /site-region >}}
 
+### Permissions
+
+By default, only users with the Datadog Admin Role can create private locations, delete private locations, and access private location installation guidelines. Admin and Standard users can view private locations, search for private locations, and assign Synthetic tests to private locations. 
+
+For more information, see the [Private locations permissions][18] documentation.
+
 ## Set up your private location
 
 ### Create your private location
 
-Go in _Synthetic Monitoring_ -> _Settings_ -> _Private Locations_ and click **Add Private Location**:
+In **Synthetic Monitoring** > **Settings** > **Private Locations**, click **Add Private Location**.
 
 {{< img src="synthetics/private_locations/add_pl.png" alt="create a private locations"  style="width:100%;">}}
 
-**Note**: Only **Admin** users can create private locations.
+**Note**: By default, only Admin users can create private locations. 
+
+Enable read and write permissions with `synthetics_private_location_read` and `synthetics_private_location_write`. Creating and modifying custom roles is an opt-in Enterprise feature. To get this feature enabled on your account, contact [Datadog support][17].
```

Comment:
Same comment as above, let's get rid of this!

Reply:
```suggestion
```